src/commcare_cloud/ansible/roles/datadog/vars/main.yml

```diff
@@ -1,7 +1,7 @@
 datadog_custom_dir: "{{ cchq_home }}/datadog"
 datadog_parsers_repo: https://github.com/dimagi/datadog-parsers
 datadog_parsers_dest: "{{ datadog_custom_dir }}/datadog-parsers"
-datedog_parsers_version: b46f25b72ed7a98d012a17ea846c3d10c418f158
+datedog_parsers_version: 4bae5c9662f7ff61993722cb398d574b89a2c89d
```

Comment:
this should be updated to the final version after https://github.com/dimagi/datadog-parsers/pull/38 and https://github.com/dimagi/datadog-parsers/pull/39

Reply:
```suggestion
datedog_parsers_version: 21fe2ed20b423669ec97cf93733b6d7ed429467b
```